packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -628,18 +628,32 @@ export async function afterTests(
     await compass.capturePage(filename);
   }
 
-  try {
-    await compass.stop(test);
-  } catch (err) {
-    debug('An error occurred while stopping compass:');
-    debug(err);
+  let timeoutId;
+  const timeoutPromise = new Promise<void>((resolve) => {
+    timeoutId = setTimeout(() => {
+      console.error('It took too long to close compass');
+      resolve();
+    }, 30000);
+  });
+
+  const closePromise = (async function close(): Promise<void> {
     try {
-      // make sure the process can exit
-      await compass.browser.deleteSession();
-    } catch (_) {
-      debug('browser already closed');
+      await compass.stop(test);
+    } catch (err) {
+      debug('An error occurred while stopping compass:');
+      debug(err);
+      try {
+        // make sure the process can exit
+        await compass.browser.deleteSession();
+      } catch (_) {
+        debug('browser already closed');
+      }
     }
-  }
+    clearTimeout(timeoutId);
+    return;
+  })();
+
+  return Promise.race([timeoutPromise, closePromise]);
```

Comment:
For some reason afterAll for some tests (where it just calls this function that closes compass) often times out after 120 seconds. Even though I can see this reach the end..

Figured I can just time it out because we're already suppressing all errors that can happen while we close compass anyway.

The reason we suppress errors is because you can't close compass if it already crashed which is the most likely reason for this stuff to fail. And we don't want to get into a situation where the entire e2e test run has to time out before it fails. Rather fail fast and with the real error, not some compounding error's eventual fallout.